includes/widgets/class-amp-widgets.php

```diff
@@ -0,0 +1,63 @@
+<?php
+/**
+ * Register widgets, and add actions.
+ *
+ * @package AMP
+ */
+
+/**
+ * Register the widgets.
+ */
+class AMP_Widgets {
+
+	/**
+	 * Add the actions.
+	 *
+	 * @return void.
+	 */
+	public function init() {
+		add_action( 'widgets_init', array( $this, 'register_widgets' ) );
+		add_action( 'show_recent_comments_widget_style', '__return_false' );
```

Comment:
Actually `remove_filter` here. Nevertheless, should this be added inside of a subclass of `WP_Widget_Recent_Comments`?

Reply:
Hi @westonruter,
Thanks for reviewing this. That's a good point to move this into a subclass of `WP_Widget_Recent_Comments`. How does [this subclass look](https://github.com/Automattic/amp-wp/pull/870/commits/be58d0243f1ef4e3533e853ff6bb436231104d4f#diff-c5a7c81e8eaa3508f2f9881ca24ccd43R20)? It might be better if I avoided adding a filter on the `__construct()` method, though.


>Actually remove_filter here

I initially used `add_action` instead of `add_filter`:
```
add_action( 'show_recent_comments_widget_style', '__return_false' );
```
But wouldn't this be enough to [prevent outputting the styling](https://github.com/WordPress/wordpress-develop/blob/master/src/wp-includes/widgets/class-wp-widget-recent-comments.php#L53):
```
add_filter( 'show_recent_comments_widget_style', '__return_false' );
```

Reply:
The subclass looks good.